docs/validation.md

```diff
@@ -1,28 +0,0 @@
-# Validation
```

Comment:
Do we want to scrap this without replacement?

Reply:
and that is moved here: https://github.com/sse-secure-systems/connaisseur/blob/docs/documentation-2.0/docs/basics.md#validation
author will have to make the current version into the main text.

Reply:
No. This is a detailed description of how it works for notaryv1, and hence is and should not be in basics.md